app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(
+        self,
+        name: str,
+        appetite: int,
+        is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print("Hello, I'm", self.name)
+
+    def feed(self) -> str:
+        if self.is_hungry:
+            print("Eating", self.appetite, "food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, 3, is_hungry)
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, 7, is_hungry)
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: List[Animal]) -> int:
+    total_food_points = sum(animal.feed() for animal in animals)
+    return total_food_points
```

Comment:
immediately write the answer do not create additional variables

Reply:
Done